orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/CompletedBakeTask.groovy

```diff
@@ -40,7 +40,11 @@ class CompletedBakeTask implements Task {
     def bakeStatus = stage.context.status as BakeStatus
     def bake = bakery.lookupBake(region, bakeStatus.resourceId).toBlocking().first()
     // This treatment of ami allows both the aws and gce bake results to be propagated.
-    def results = [ami: bake.ami ?: bake.imageName, imageId: bake.ami ?: bake.imageName, artifact: bake.artifact ?: new Artifact()]
+    def results = [
+      ami: bake.ami ?: bake.imageName,
+      imageId: bake.ami ?: bake.imageName,
+      artifacts: bake.artifact ? [bake.artifact] : []
```

Comment:
It's a little worrying that the bake itself can only emit one artifact... While this works here, we may want to address this in Rosco going forward.

Reply:
Yes, I noticed that as well; I'll look into allowing a bake to produce more than one artifact when I dig more into Rosco.